eeg_eyetracking_parser/_eeg_preprocessing.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np
+import mne
+import os
+from . import trial_trigger
+from mne.preprocessing import ICA
+import autoreject as ar
+import json
+import matplotlib.pyplot as plt
+
+
+def rereference_channels(raw, ref_channels=['A1', 'A2']):
+    """
+    Rereference EEG data to the mastoids
+    """
+    raw.set_eeg_reference(ref_channels=ref_channels)
+    raw.drop_channels(ref_channels)
+    return raw
```

Comment:
Functions that modify the `raw` in place shouldn't return the `raw`. This behavior should be clearly described in the docstring.